back/spec/acceptance/initiatives_spec.rb

```diff
@@ -157,6 +157,71 @@
       expect(json_response[:data].filter_map { |d| d[:relationships][:user_reaction][:data] }.first[:id]).to eq reaction.id
       expect(json_response[:included].pluck(:id)).to include reaction.id
     end
+
+    context 'when approval_pending and approval_rejected initiatives exist' do
```

Comment:
I think these tests should be in the new `initiative_policy_spec.rb` file (it doesn't exist atm). It's not a requirement, but it should be pretty easy to change.

Sometimes `document: false` is a good sign the spec doesn't belong here.